pages/archive/index.vue

```diff
@@ -0,0 +1,22 @@
+<script setup lang="ts">
+const { t } = useI18n();
+toolbarTitleStore().setReactiveToolbarTitle(() => t("nav.archive"));
+
+const { data: years } = useYearList();
+</script>
+
+<template>
+  <div>
+    <div class="h-3"></div>
+    <div class="columns-2 md:columns-3 xl:columns-4 2xl:columns-5">
+      <NuxtLink
+        v-for="(year, i) in years?.toReversed()"
+        :key="`year${i}`"
```

Comment:
Using the index as the key here has no effect, as indexes might easily change (reordering, adding items, etc).
We should use some kind of unique identifier if possible. 

https://vuejs.org/guide/essentials/list.html#maintaining-state-with-key

Reply:
done